src/app.ts

```diff
@@ -1,18 +1,19 @@
 import express from 'express';
 import cors from 'cors';
 import { HOSTNAME, PORT } from './config';
-import { groupRouter, userGroupRouter, userRouter } from './routes';
+import { groupRouter, loginRouter, userGroupRouter, userRouter } from './routes';
+import { checkToken, errorHandlerMiddleware, loggerMiddleware } from './middlewares';
 import { Logger } from './components';
-import { errorHandlerMiddleware, loggerMiddleware } from './middlewares';
 import { HttpStatusCode } from './types';
 
 const app = express();
 app.use(cors());
 app.use(loggerMiddleware);
 
-app.use('/user/', userRouter);
-app.use('/group/', groupRouter);
-app.use('/user-group/', userGroupRouter);
+app.use('/login/', loginRouter);
+app.use('/user/', checkToken, userRouter);
```

Comment:
I assume in most backend apps most of you routes will require token check and only couple of them will be able to work without it. So how about updating all this stuff with creating list of exclusions which do not require token?